lib/bench/contains.bench.js

```diff
@@ -16,8 +16,8 @@ module.exports = {
     'contains(c, cs)': function() {
       return contains(randomLowerAlpha(), alphabet);
     },
-    'native indexOf': function() {
-      return nativeIndexOf(randomLowerAlpha(), alphabet);
+    'native includes': function() {
```

Comment:
I think it makes more sense to call this 'native contains' even if the native function we use is not called contains (either way definitely better than `indexOf` from before).